test/e2e/auth/projected_clustertrustbundle.go

```diff
@@ -56,7 +56,8 @@ const (
 	noSignerKey       = "no-signer"
 )
 
-var _ = SIGDescribe(feature.ClusterTrustBundle, feature.ClusterTrustBundleProjection, func() {
+// TODO: running the tests in parallel should be possible
```

Comment:
Curious on why this change? Were tests not okay before?

Reply:
Hunh, I think this was a merge/rebase that I missed. thanks! should be fixed now.